README.md

```diff
@@ -117,6 +117,17 @@ Then you can install dependencies with:
 npm install
 ```
 
+> ### ⚠️ Working on a mac with an M1 chip?
+> 
+> You will need to set the `M1` environment variable before installing dependencies:
```

Comment:
I guess it is implied, but this wording sounds like the `M1` setting is only needed for `npm install`. It needs to be in effect for any `npm run` commands later as well.

This is a potential pitfall if you work with multiple shells (e.g. to keep separate scrollback buffers, or because `npm run dev` blocks the shell while the app is running). You need to set `export M1=1` in each shell you want to run `npm run ...` commands.

Reply:
That's a good call-out. I'll work on rewording this so that it's more clear that the environment variable needs to be set before running any npm scripts

Reply:
I updated to explain that the environment variable needs to be set before running npm scripts as well.